README.md

```diff
@@ -1,5 +1,48 @@
+**![](https://img.shields.io/badge/-microverse-yellow)**
+
 # Basic HTML Page Structure (Youtube Page)
 
 > This is the first project on CSS/HTML curriculum in microverse course
 
+# Screenshot
+
+![screenshot](/src/screenshot_1.png)
+
+## Participants
+
+- 🤓 **Victor Pacheco**
+- GitHub: [@elberthcorniell](https://github.com/vichuge)
+- 😎 **Abdo Amin**
+- GitHub: [@Abdelrhman-Amin](https://github.com/AbdelrhmanAmin)
+- Twitter: [@Abdo-Amin](https://twitter.com/AbdoAmi60489112)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/abdo-amin-ab786a1b0/)
+
+## Built
+
+- HTML
+- CSS
+
+## Live Demo
+
+[Live_Demo](https://raw.githack.com/vichuge/BasicHTMLPageStructure/feature-branch/index.html)
+
+## SETUP
+
+    1- Open CMD
+    2- git clone https://github.com/vichuge/BasicHTMLPageStructure.git
+    3- Type " Code ." on CMD to open VSCode
+    4- Open with LIVE Server.
+    5- SMILE! :)
+
+## Usage
+
+      This project is made using HTML & CSS.
+      You don't need any tools to run it, Just open the index.html in your favorite browser and it's done.
+
+## Contribuiting
```

Comment:
Instead of writing `Contribuiting`, you should write `Contributing` :point_right: that's a typo, fix it